CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [1.1.8](https://github.com/ably/ably-php/tree/1.1.8) (2022-07-27)
+
+[Full Changelog](https://github.com/ably/ably-php/compare/1.1.7...1.1.8)
+
+**Implemented enhancements:**
+
+- Add support to get channel lifecycle status  [\#155](https://github.com/ably/ably-php/issues/155)
+
+**Merged pull requests:**
+
+- Add support for MsgPack [\#161](https://github.com/ably/ably-php/pull/161) ([AmirQSD](https://github.com/AmirQSD))
```

Comment:
I think you should add this one to the implemented enhancements section too

Reply:
OK, keep it in both places then? @owenpearson

Reply:
yeah exactly 👍